index.d.ts

```diff
@@ -40,7 +40,17 @@ export type ResponseType =
   | 'text' 
   | 'stream'
 
-export interface AxiosRequestConfig {
+export type ResponseEncoding =
+  | 'ascii'
+  | 'utf8'
+  | 'utf16le'
+  | 'ucs2'
+  | 'base64'
+  | 'latin1'
+  | 'binary'
+  | 'hex'
```

Comment:
I'd like keep with [Node.js API documents](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end) temporarily and not limit to some certain types.